client/asset/eth/eth.go

```diff
@@ -4383,19 +4385,107 @@ func runSimnetMiner(ctx context.Context, log dex.Logger) {
 	}()
 }
 
-// GetGasEstimates gets estimates useful for populating dexeth.Gases fields.
-// Initiation and redeeem transactions with 1, 2, ... , and n swaps per
-// transaction will be sent, for a total of n * (n + 1) / 2 total swaps in 2 * n
-// transactions. If this is a token, and additional 1 approval transaction and 1
-// transfer transaction will be sent. The transfer transaction will send 1 atom
-// to a random address (with zero token balance), to maximize gas costs. This
-// atom is not recoverable.
-// If you run this function with insufficient or zero ETH and/or token balance
-// on the seed, the function will error with a message indicating the amount of
-// funding needed to run.
-func GetGasEstimates(ctx context.Context, net dex.Network, assetID, contractVer uint32, maxSwaps int, credentialsPath string, log dex.Logger) error {
+type getGas byte
+
+// GetGas provides access to the gas estimation utilities.
+var GetGas getGas
+
+// ReadCredentials reads the credentials for the network from the credentials
+// file.
+func (getGas) ReadCredentials(credentialsPath string, net dex.Network) (addr, provider string, err error) {
```

Comment:
Interesting.  Using methods with no receiver variables to sorta make a namespace for these functions rather than prefix `GetGas*` to all the vanilla functions?

Reply:
Exactly.